code/src/containers/forms/new-product-form/NewProductForm.scss

```diff
@@ -0,0 +1,138 @@
+@import "@design-system";
+
+.product-form {
+  display: grid;
+  grid-template-columns: calc(30% - 10px) calc(70% - 10px);
+  grid-gap: spacing(5);
+  box-sizing: border-box !important;
+  margin-bottom: spacing(5);
+
+  & div {
+    box-sizing: border-box !important;
```

Comment:
Add comments to `!important`

Reply:
Thanks, removed unnecessary `!important` and added comments to others `!important`